File: TEO/Mav.UI/Mav.Ui/src/app/models/admin/organisation.view.model.ts
import { LicenceViewModel } from "./licence.view.model";
import { OrganisationTypeViewModel } from "./organisation-type.view.model";

export class OrganisationViewModel {
    public id: number;

    public typeID: number;

    public licenceID: number;

    public organisationName: string;

    public addressLine1: string;

    public addressLine2: string;

    public addressLine3: string;

    public mainContactName: string;

    public mainContactEmail: string;

    public leadFinancialName: string;

    public leadFinancialEmail: string;

    public leadFinancialNumber: string;

    public centreLimit: number;

    public licence: LicenceViewModel;

    public organisationType: OrganisationTypeViewModel;
}

File: TEO/Mav.Models/ViewModels/OrganisationViewModel.cs
using Mav.Data.Entities;
using Mav.Models.SearchModels;
using System;
using System.Collections.Generic;
using System.Text;

namespace Mav.Models.ViewModels
{
    public class OrganisationViewModel
    {
        public int ID { get; set; }

        public int TypeID { get; set; }

        public int LicenceID { get; set; }

        public string OrganisationName { get; set; }

        public string AddressLine1 { get; set; }

        public string AddressLine2 { get; set; }

        public string AddressLine3 { get; set; }

        public string MainContactName { get; set; }

        public string MainContactEmail { get; set; }

        public string LeadFinancialName { get; set; }

        public string LeadFinancialEmail { get; set; }

        public string LeadFinancialNumber { get; set; }

        public int CentreLimit { get; set; }

        public LicenceViewModel Licence { get; set; }

        public OrganisationTypeViewModel OrganisationType { get; set; }
    }
}
